src/Player.js

```diff
@@ -168,8 +168,8 @@ class Player extends EventEmitter {
 
             return song;
         }
-        catch (err = {}) {
-            this.emit('error', err.message || err, message);
+        catch (err) {
+            this.emit('error', err ? err.message : err, message);
```

Comment:
We should use `err.message ?? err`, as if err is a string it will still log `undefined`.